lib/trie/version.go

```diff
@@ -7,8 +7,12 @@ import (
 	"errors"
 	"fmt"
 	"strings"
+
+	"github.com/ChainSafe/gossamer/lib/common"
 )
 
+const V1MaxValueSize = common.HashLength
```

Comment:
curious what others think, but im kind of in favor of avoiding using common types where we can

Reply:
But totally fine to keep here! Just airing my beef with the common pkg

Reply:
I would just make this 
```suggestion
const V1MaxValueSize = 32
```